docs/BreakingChanges/126 GZipStream does not flush until it is disposed.md

```diff
@@ -0,0 +1,30 @@
+## 126: GZipStream does not flush until it is disposed
+
+### Scope
+Minor
+
+### Version Introduced
+4.5
+
+### Source Analyzer Status
+Planned
+
+### Change Description
+Due to changes in the underlying compression library, a GZipStream will now not flush its buffers until it is disposed.
+
+- [ ] Quirked
+- [ ] Build-time break
+
+### Recommended Action
+When using a GZipStream to write to a MemoryStream or other stream that will be read from within the same process, make sure that the GZipStream has been disposed before attempting to read from the output stream. Note that calling `Flush` is insufficient. Because the GZipStream is being disposed, it may be necessary (depending on specific code patterns) to use the GZipStream constructor that takes a boolean specifying to leave its underlying stream open when the GZipStream object is disposed.
+
+### Affected APIs
+* `M:System.IO.Compression.GZipStream.Flush`
+
+[More information](http://www.mistergoodcat.com/post/Working-around-a-bug-in-MicrosoftBclCompression)
```

Comment:
This is an interesting break because it's the only one (I think) that has a more info link pointing somewhere other than MSDN, Connect, or SO. I think that's ok - the blog we link here does describe the issue and workaround pretty well - but there's a danger the site could vanish in the future.

Probably, we should have some automation to make sure that More Info links are valid and make a work item to create an MS-owned blog post about this issue (so that we can stop pointing at MisterGoodCat.com)